coa-helm/templates/cromwell.yaml

```diff
@@ -157,11 +157,11 @@ data:
     }
 
     database {
-      db.url = "jdbc:mysql://mysqldb/{{ .Values.db.name }}?useSSL=false&rewriteBatchedStatements=true&allowPublicKeyRetrieval=true"
-      db.user = {{ .Values.db.user | quote }}
-      db.password = {{ .Values.db.password | quote }}
-      db.driver = "com.mysql.cj.jdbc.Driver"
-      profile = "slick.jdbc.MySQLProfile$"
+      db.url = "jdbc:postgresql://coa-postgres:{{ .Values.postgres.port }}/{{ .Values.postgres.cromwell.dbname }}?useSSL=false&rewriteBatchedStatements=true&allowPublicKeyRetrieval=true"
```

Comment:
`coa-postgres` is the service name. I don't think it should be set as a value, but it probably should be defined dynamically with a template value (that's how service names are defined elsewhere in these charts)

Reply:
it's worth pointing out here that this service name -- `coa-postgres` -- violates some conventions I have been trying to impose on these charts. Ideally it would be `coa-postgres-service` (to fit the pattern `<app fullname>-<component name>-<kubernetes object>`). But doing so would require changing parts of `cromwell-helm`, which I hesitate to do in this PR. 

This PR is intended only to get Postgres working with the Azure app. I didn't want to put too many cleanup changes here.

Reply:
+1 to changing that in a follow up